### PR TITLE
Fix metrics

### PR DESCRIPTION
**Pull request by Half-Shot**

### CODE REVIEW

File: src/bridge/IrcBridge.ts [hunk 1]
```diff
@@ -198,11 +198,9 @@ export class IrcBridge {
         const usingRemoteMetrics = !!this.config.ircService.metrics.port;
 
         const metrics = this.bridge.getPrometheusMetrics(!usingRemoteMetrics, registry);
-
+        let metricsUrl = `${this.config.homeserver.bindHostname || "0.0.0.0"}:${this.config.homeserver.bindPort}`;
         if (this.config.ircService.metrics.port) {
-            log.info(
-            `Started metrics on http://${this.config.ircService.metrics.host}:${this.config.ircService.metrics.port}`
-            );
+            metricsUrl = `${this.config.ircService.metrics.host}:${this.config.ircService.metrics.port}`;
```
Comment (jaller94) [inline src/bridge/IrcBridge.ts:203]: I think `metrics`, `host` and `port` are all optional, aren't they?

File: src/bridge/IrcBridge.ts [hunk 2]
```diff
@@ -198,11 +198,9 @@ export class IrcBridge {
         const usingRemoteMetrics = !!this.config.ircService.metrics.port;
 
         const metrics = this.bridge.getPrometheusMetrics(!usingRemoteMetrics, registry);
-
+        let metricsUrl = `${this.config.homeserver.bindHostname || "0.0.0.0"}:${this.config.homeserver.bindPort}`;
```
Comment (jaller94) [inline src/bridge/IrcBridge.ts:201]: Is bindPort required?

File: src/bridge/IrcBridge.ts (hunk 1 above)
Comment (Half-Shot) [reply src/bridge/IrcBridge.ts:203]: We check metrics further up, port is checked but the hostname could be missing

**Comment by Half-Shot** [comment]
Tried to unmangle the url stuff

**Pull request merged**